packages/kbn-generate/src/commands/codeowners_command.ts

```diff
@@ -30,6 +30,9 @@ const GENERATED_TRAILER = `
 
 # Design (at the bottom for specificity of SASS files)
 **/*.scss  @elastic/kibana-design
+
+# Changes to translation files should not ping code reviewers
```

Comment:
I don't think this is valid syntax (highlighted at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax)

I believe we need to set an empty code owners line like 

```
# Changes to translation files should not ping code reviewers
x-pack/plugins/translations/translations 
```

Reply:
oh! I knew about negation, but didn't know about the empty assignment. I like the assertive way better. Pushed in [`fa838a1` (#150678)](https://github.com/elastic/kibana/pull/150678/commits/fa838a14a954e3f104329bda4e8a006745bbd14e)

Reply:
> @spalger do you think in the future we should accommodate for this behaviour by supporting an object for the owners .jsonc field where we can set exclusion folders/files?

FWIW, I think we could move these `translation/translation` files to a package with no owners. However, that involves modifying a fair amount of scripts in our localization pipeline. I'm happy to create a follow-up issue with that goal if it makes sense to you.

Reply:
I could imagine something in the package manifest that describes owner information in more detail, but I think it sounds pretty complicated and I like the idea of an ownerless package more. "One more package" feels like a better solution here

Reply:
I've created https://github.com/elastic/kibana/issues/150753 as a follow-up to create the ownerless package (and adapt all the scripts depending on these files).

Is it OK to merge this PR while we tackle that?

Reply:
For sure 👍